scripts/CEE/get-kafka-instance-state/get-kafka-instance-state.sh

```diff
@@ -42,6 +42,22 @@ function main() {
   oc exec -n "${resource_namespace}" "${kafkaStatefulSet}" -c kafka -- sh /opt/kafka/bin/kafka-acls.sh \
     --bootstrap-server localhost:9096 --list > "${inspectDir}"/kafka-acls.txt 2>&1
 
+  oc exec -n "${resource_namespace}" "${kafkaStatefulSet}" -c kafka -- sh /opt/kafka/bin/kafka-transactions.sh \
+    --bootstrap-server localhost:9096 list > "${inspectDir}"/kafka-transactions-list.txt 2>&1
+
+  oc exec -n "${resource_namespace}" "${kafkaStatefulSet}" -c kafka -- sh /opt/kafka/bin/kafka-transactions.sh \
+    --bootstrap-server localhost:9096 list > "${inspectDir}"/kafka-transactions-list.txt 2>&1
+
+  oc exec -n "${resource_namespace}" "${kafkaStatefulSet}" -c kafka -- sh -c \
+    "awk 'NR > 1 { print \$1 }' ${inspectDir}/kafka-transactions-list.txt 2>&1 | xargs -L 1 ./bin/kafka-transactions.sh --bootstrap-server localhost:9096 describe --transactional-id \$@ > ${inspectDir}/kafka-transactions-describe.txt 2>&1"
```

Comment:
Does `${inspectDir}/kafka-transactions-describe.txt` filename need the transaction-id interpolated into it to make it unique.

Reply:
Either that, or each invocation should append to the file rather than overwriting it.